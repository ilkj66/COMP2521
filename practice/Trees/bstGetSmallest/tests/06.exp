Enter the preorder traversal of the BST: 
BST:

X

bstreeGetSmallest returned: NULL
