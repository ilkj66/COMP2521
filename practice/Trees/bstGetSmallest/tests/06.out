Enter the preorder traversal of the BST: 
BST:

X

bstGetSmallest returned: NULL
